lib/widgets/mxc_image.dart

```diff
@@ -233,9 +233,16 @@ class _MxcImageState extends State<MxcImage>
 
   Widget placeholder(BuildContext context) =>
       widget.placeholder?.call(context) ??
-      const Center(
-        key: Key(placeholderKey),
-        child: CircularProgressIndicator.adaptive(),
+      SizedBox(
+        width: widget.width ?? 256,
+        height: widget.height ?? 256,
```

Comment:
@sherlockvn please review it

Reply:
Why don't you set `placeholder` in where you use, instead of change the `MxcImage`

Reply:
deleted